test-package.bats

```diff
@@ -36,6 +36,15 @@ DOC
 }
 
 
+@test "package setup" {
```

Comment:
Please fix the docstring
Suggest to name the test `test local setup` and change the docstring correspondingly